hera_cal/tests/test_tempcal.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 the HERA Project
+# Licensed under the MIT License
+
+from __future__ import print_function, division, absolute_import
+
+import pytest
+import numpy as np
+import sys
+import os
+import glob
+import six
+import copy
+from pyuvdata import UVData
+
+from .. import utils, io, apply_cal
+from ..tempcal import TempCal
+from ..data import DATA_PATH
+
+
+def test_tempcal():
```

Comment:
You should break your tests into multiple discrete tests, ideally one per function.

Reply:
the load is not terribly fast, so I can probably chunk it into two or three separate functions, but don't want to do more than that.